rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/InterpretedLanguageContributions.java

```diff
@@ -370,8 +403,26 @@ public CompletableFuture<SummaryConfig> getOndemandSummaryConfig() {
     public InterruptibleFuture<@Nullable IValue> executeCommand(String command) {
         logger.debug("executeCommand({}...) (full command value in TRACE level)", () -> command.substring(0, Math.min(10, command.length())));
         logger.trace("Full command: {}", command);
-        return InterruptibleFuture.flatten(parseCommand(command).thenCombine(commandExecutor,
-            (cons, func) -> EvaluatorUtil.<@Nullable IValue>runEvaluator("executeCommand", eval, ev -> func.call(cons), null, exec, true, client)
+
+        return InterruptibleFuture.flatten(parseCommand(command).thenCombine(
+            commandExecutor,
+            (cons, func) -> {
+                
+                if (func == null) {
+                    logger.info("Command is being executed without a registered command executor; for language " + name);
+                    throw new RuntimeException("No command executor registered for " + name);
+                }
+
+                return EvaluatorUtil.<@Nullable IValue>runEvaluator(
```

Comment:
is there a specific reason this is now a multiline call? I think it doesn't match the style of the other functions in this class?

Reply:
I had a lot of benefit from putting breakpoints on the individual lines, so I'm leaving it at that for now.